classes/PodsAdmin.php

```diff
@@ -2441,4 +2444,197 @@ function configuration( $pod = null, $full_field_info = false ){
 
 	}
 
+    /**
+     * Build UI for extending REST API, if makes sense to do so.
+     *
+     * @todo maybe disable if default routes don't exist since WP < 4.5 and plugin isn't installed?
+     *
+     * @since 2.5.6
+     *
+     * @access protected
+     */
+    protected function rest_admin() {
+        if( function_exists( 'register_rest_field' ) ) {
```

Comment:
I couldn't locate a register_rest_field function anywhere.  Is this supposed to be register_api_field?  Maybe this is an old (or potentially new) function?